### ISSUE TITLE
Add support for thread author shown in thread

### ISSUE THREAD

**Issue by Eldenroot** [benoStrict/MyBB_lightAVATARS#1]

**Comment by benoStrict** [comment]
Hello.
Thanks for suggestions. 
Can you describe this idea?

**Comment by Eldenroot** [comment]
flavatar from Tomm - it supports a variable for show thread author in a thread

**Comment by benoStrict** [comment]
If you think about this https://zapodaj.net/86ab58331d1b4.png.html
It is available in the latest version. Just use
{+LIGHTAVATARS+}custom1|{$post['uid']}{+ENDofBLOCK+}
You can also add your own settings.
At the moment guide is available only in Polish.
I will write the English version of the guide soon.
https://mybboard.pl/temat-lightavatars-kilka-mniej-znanych-funkcji_69001

**Comment by Eldenroot** [comment]
Yeah, English version for users with little knowledge of coding would be nice.

I mean this:
<span>Started by <a href='{$flp_avatar['profile']}' title='View {$flp_avatar['username']}'s Profile'>{$flp_avatar['username']}</a></span> - https://github.com/tommm/flpavatar

**Comment by benoStrict** [comment]
Do you mean to split {+}zzz|yyy{-} variable into parts with the possibility to modifying the html template by user in the ACP?
Sorry, but my English isn't very good.
Screenshot https://zapodaj.net/32047f66c3f4d.png.html
